fieldservice_maintenance/models/maintenance_request.py

```diff
@@ -24,23 +28,26 @@ def create(self, vals):
             fsm_order_type = self.env["fsm.order.type"].search(
                 [("internal_type", "=", "maintenance")], order="id desc", limit=1
             )
-            if not fsm_equipment.current_location_id.id:
-                raise UserError(
-                    _("Missing current location on FSM equipment %s")
-                    % fsm_equipment.name
-                )
-            fsm_order_id = self.env["fsm.order"].create(
-                {
-                    "type": fsm_order_type.id,
-                    "equipment_id": fsm_equipment.id,
-                    "location_id": fsm_equipment.current_location_id.id,
-                    "request_id": request.id,
-                    "description": request.description,
-                    "request_early": request.schedule_date,
-                    "scheduled_date_start": request.schedule_date,
-                    "priority": request.priority,
-                }
-            )
-
-            request.fsm_order_id = fsm_order_id
+            if request.create_fsm_order:
+                if not fsm_equipment.current_location_id.id:
+                    _logger.info(
+                        _(
+                            "Missing location on fsm_equipment %s. fsm_order was not created"
+                            % fsm_equipment.id
```

Comment:
@PicchiSeba What do you think of adding this message to the chatter of the maintenance request as well? The user does not see the logs.

Reply:
makes sense. thank you

Reply:
Hi, sorry for being late. I applied the suggestion